src/util/bip32.rs

```diff
@@ -57,8 +57,8 @@ pub struct ExtendedPrivKey {
     pub parent_fingerprint: Fingerprint,
     /// Child number of the key used to derive from parent (0 for master)
     pub child_number: ChildNumber,
-    /// Secret key
-    pub secret_key: SecretKey,
+    /// Private key
+    pub secret_key: PrivateKey,
```

Comment:
Perhaps we can rename this to `private_key`?

Reply:
Sure. I considered this but wanted to minimize the diff. But I guess we'll be tripping over this for years if I don't change it ..